roles/local_repo_setup/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+# vim: set ft=ansible:
+#
+# This role sets up local repo of the booted deployment
+#
+- name: Fail if branch_name is undefined
+  fail:
+    msg: "branch_name is undefined"
+  when: branch_name is undefined
+
+- name: Get rpm-ostree status output
+  command: rpm-ostree status --json
+  register: ros
+
+- name: Convert rpm-ostree output to JSON
+  set_fact:
+    ros_json: "{{ ros.stdout|from_json }}"
+
+- name: Set current checksum
+  set_fact:
+    checksum: "{{ ros_json['deployments'][0]['checksum'] }}"
+  when: ros_json['deployments'][0] is defined and
+        ros_json['deployments'][0]['booted']
+
+- name: Set current checksum
```

Comment:
Yeah, I would definitely like a cleaner way to do this!  I'll look around to see if I can find anything.